docs/infrastructure/microk8s-quickstart.md

```diff
@@ -0,0 +1,348 @@
+
+### Helm and tiller
+
+Two executables will get installed onto the cluster as part of this process: `helm` and `tiller`. The former is responsible for resolving the configuration of the chart to be installed, while issuing the correct install commands onto the cluster. The latter is responsible for receiving the install commands issued by `helm`, as well as managing the lifecycle of the components that have been installed. `helm` is the client facing side, while `tiller` is the server/cluster facing side.
+
+## 2. Prepare your environment
+
+Prepare your environment to set up the microk8s cluster:
+
+1. a machine running Ubuntu 18.04 LTS version.
+2. establish a remote session (ssh) onto this machine.
+
+Assuming that you are starting from a blank slate, the first step is to install microk8s. Otherwise, please jump on to the [Configuring microk8s](###Configuring-microk8s) section.
+
+## 3. Installing microk8s
+
+1. Make sure the machine is up to date and also has the `nfs-common` package installed
+   >apt update && apt install nfs-common -y
+
+2. Install microk8s
+   >sudo snap install microk8s --classic --channel=1.15/stable
+   1. We recommend following the official documentation on how to [install microk8s](https://microk8s.io/docs/).
+
+## 4. Configuring microk8s
+
+1. First, we must enable the following plugins on microk8s:
+    >sudo microk8s.enable dns && sudo microk8s.status --wait-ready
+
+    >sudo microk8s.enable storage && sudo microk8s.status --wait-ready
+
+    >sudo microk8s.enable ingress &&  sudo microk8s.status --wait-ready
+
+2. Now that these plugins have been enabled, we should restart microk8s to ensure a proper bootstrap of the cluster
+    >microk8s.stop && microk8s.start && microk8s.status --wait-ready
+
+    Further information on this subject can be [found here](https://github.com/ubuntu/microk8s/issues/493#issuecomment-498167435).
+3. The plugins are now enabled and the cluster bootstrapped. However, we must still wait for some microk8s internals (dns, http, and ingress) plugins to be ready. Failing to do so can result in pods entering a `CrashLoopBackoff` state:
+
+    >microk8s.kubectl wait -n kube-system --for=condition=Ready pod -l k8s-app=kube-dns --timeout=300s
+
+    >microk8s.kubectl wait -n kube-system --for=condition=Ready pod -l k8s-app=hostpath-provisioner --timeout=300s
+
+    >microk8s.kubectl wait -n ingress --for=condition=Ready pod -l name=nginx-ingress-microk8s --timeout=300s
+
+4. Export your microk8s cluster configuration to the local kubeconfig file
+    >microk8s.config > ~/.kube/config
+
+After runing these commands, we have ensured that dns, http, and nginx ingress are enabled and working properly inside the cluster.
+
+## 5. Installing Codacy
+
+### 1. Helm and Tiller
```

Comment:
I would move this section to **Configuring microk8s**, as we're still taking care of the prerequisites and not installing Codacy yet.

Reply:
But helm is not microk8s specific. It is required to install the codacy chart :/

Reply:
I agree with @pauloribeiro-codacy . This looks a lot like setting up prerequisites to then install codacy. Microk8s is a prerequisite and Helm is another.